<launch>
  <node pkg="iai_rosbag_utils" type="tf_repeater" name="tf_repeater_shoulder_kinect_link">
  	<param name="parent_frame" value="/base_link" />
  	<param name="child_frame" value="/shoulder_kinect_link" />
    <remap from="~in_topic" to="/tf_real"/>
    <remap from="~out_topic" to="/tf"/>
  </node>
  <node pkg="iai_rosbag_utils" type="tf_repeater" name="tf_repeater_shouler_kinect_rgb_frame">
  	<param name="parent_frame" value="/shoulder_kinect_link" />
  	<param name="child_frame" value="/shoulder_kinect_rgb_frame" />
    <remap from="~in_topic" to="/tf_real"/>
    <remap from="~out_topic" to="/tf"/>
  </node>
</launch>
